krux_marathon_api/cli.py

```diff
@@ -60,6 +62,16 @@ def add_cli_arguments(self, parser):
             help='Marathon server port. ',
         )
         group.add_argument(
+            '--username',
+            default=os.getenv('MARATHON_HTTP_USERNAME'),
+            help='Marathon server username (if needed)'
```

Comment:
Since these are optional parameters, can you also say what are the default values in the help doc? Refer to `krux-boto` if needed.